backend/controllers/users.js

```diff
@@ -29,5 +28,8 @@ export const UsersController = {
       // returns a body containing the post object and token string
       res.status(200).json({ user })
     })
+  },
+  CreateProfile: (req, res) => {
```

Comment:
if this doesn't do anything, can we remove it? Charlie and i created a different route for add/editing profiles